docker/spec/dependabot/docker/file_parser_spec.rb

```diff
@@ -556,7 +556,7 @@
     context "with a non-standard filename" do
       let(:dockerfile) do
         Dependabot::DependencyFile.new(
-          name: "custom-name",
+          name: "custom-dockerfile-name",
```

Comment:
I was confused by the initial test here. A 'dockerfile' with a name 'custom-name' should never have been picked up by the file_fetcher code which, if I understand correctly, is what will feed the parser code. So the string 'dockerfile' should be in the name somewhere to be picked up and that is what this change is doing and what the larger code change in general uncovered.

Could you confirm?